src/app/main-overview/components/ZoneControlPlanesDetails.vue

```diff
@@ -8,6 +8,35 @@
     disable-sorting
     hide-pagination-when-optional
   >
+    <template
+      v-if="props.zoneOverviews.length === 0"
+      #empty-state
+    >
+      <EmptyBlock>
```

Comment:
As CTA are meant to be systematic. Would make sense to always have this in the AppCollection?

You'd only take conf for:

- title
- desc
- button content
- where the button gets us

I feel like it'd be easier to adopt than all of this for each of the lists

Reply:
I agree. I turned empty state usage for AppCollection into props and changed the two tables on the overview to use AppCollection so they can make use of the same mechanism.